test/inductor/test_torchinductor.py

```diff
@@ -14824,6 +14824,27 @@ def f(x, y):
             compiled_out = f_compiled(x, y)
             self.assertEqual(compiled_out, f(x, y))
 
+        @torch._inductor.config.patch("graph_partition", True)
+        def test_graph_partition_symint_cat_backward(self):
+            def f(x, w):
+                y = torch.cat((x, x), dim=0)
+                z = y @ w
+                return z @ z.T
+
+            compiled_f = torch.compile(f)
+
+            for shape in (2, 3):
+                torch.manual_seed(42)
+                eager_x = torch.randn(shape, 2, device="cuda")
```

Comment:
Hi, May I suggest to replace the hard code `"cuda"` with `GPU_TYPE` in this case ? Otherwise the `"cuda"` will fail on other GPUs.